lib/locales/en.ts

```diff
@@ -304,6 +309,7 @@ export const locale: Locale = {
     christopher_of_palestine_martyr: 'Saint Christopher, Martyr',
     ciaran_of_clonmacnoise_abbot: 'Saint Ciaran, Abbot',
     clare_of_assisi_virgin: 'Saint Clare, Virgin',
+    claudine_thevenet_religious: 'Saint Claudine Thévenet, Religious',
```

Comment:
Add her religious name to the key name.

```suggestion
    marie_of_saint_ignatius_claudine_thevenet_religious: 'Saint Claudine Thévenet, Religious',
```